style.css

```diff
@@ -762,7 +767,162 @@ input[type=submit]:disabled {
   height: 100vh;
 }
 
+.modal {
+  width: 100%;
+  height: 180%;
+  background-color: rgba(0, 0, 0, 0);
+  backdrop-filter: blur(6px);
+  position: absolute;
+  top: 0;
+  display: none;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal-content {
+  width: 450px;
+  height: 950px;
+  background-color: white;
+  border: 1px solid #dfe1e6;
+  opacity: 1;
+  position: relative;
+}
```

Comment:
Kindly update your popup window in the desktop version to match the figma design better🙏

Your design:
<img width="510" alt="image" src="https://github.com/YoutLone/Portfolio/assets/37116322/db3b8587-faa5-4c04-9896-d5eb2a6dca12">

Figma design:
<img width="315" alt="image" src="https://github.com/YoutLone/Portfolio/assets/37116322/8ba2cfe4-47a5-4798-b212-3122e5e6250a">

A few things to note:
- Please make your popup window fullscreen and prevent user scrolling to stop an overlap of your design
- Please reduce the padding at the bottom of the popup window
- Your popup window only looks good on certain screen sizes in your desktop version. Please make sure your popup window uses responsive widths to make sure it is responsive on all desktop screen sizes (> 768px)